source/layouts/_hero.erb

```diff
@@ -1,10 +1,10 @@
-<section class="hero-section">
+<section class="hero-section" aria-labelledby="heroHeader">
```

Comment:
```suggestion
<section class="hero-section" aria-label="Introduction to CodeLand 2021">
```

suggestion (not blocking) - I think the name of "CodeLand 2021" for this region might not be the _most_ helpful (since the whole site is about CodeLand 2021 😄 ), but in saying that I can't think of much better so feel free to take or leave the suggestion!